hardware/raspberrypi/bcm2711_bootloader_config.md

```diff
@@ -43,89 +18,92 @@ This section describes all the configuration items available in the bootloader.
 
 If 1 then enable UART debug output on GPIO 14 and 15. Configure the receiving debug terminal at 115200bps, 8 bits, no parity bits, 1 stop bit.
 
-Default: 0  
+Default: `0`  
 Version: All  
 
 ### WAKE_ON_GPIO
 
 If 1 then 'sudo halt' will run in a lower power mode until either GPIO3 or GLOBAL_EN are shorted to ground.
```

Comment:
`1` in backticks.

Reply:
Also, the meaning could do with some further explanation, e.g.:

`If 1 then once the operating system shuts down (e.g. from sudo halt), the Raspberry Pi will run in a low-power mode until either GPIO3 or GLOBAL_EN are shorted to ground.`

Reply:
I think it's self-explanatory